File: benchmark/fibonacci/fib.c
#include <stdio.h>

int fib(int n) {
    if (n < 2) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

int main() {
    int n = 30;
    int result = fib(n);
    printf("fib(%d) = %d\n", n, result);
    return 0;
}

File: benchmark/fibonacci/fib.lua
local function fib(n)
    if n < 2 then
        return n
    end
    return fib(n - 1) + fib(n - 2)
end

local n = 35
local result = fib(n)
print(string.format("fib(%d) = %d", n, result))
